packages/resource-deployment/scripts/setup-cosmos-db.sh

```diff
@@ -81,7 +81,7 @@ function setupCosmos() {
         cosmosSetupProcesses=(
             "createCosmosCollection \"scanRuns\" \"$onDemandScannerDbName\" \"2592000\" \"20000\""        # 30 days
             "createCosmosCollection \"scanBatchRequests\" \"$onDemandScannerDbName\" \"604800\" \"2000\"" # 7 days
-            "createCosmosCollection \"scanRequests\" \"$onDemandScannerDbName\" \"604800\" \"20000\""     # 7 days
+            "createCosmosCollection \"scanRequests\" \"$onDemandScannerDbName\" \"604800\" \"10000\""     # 7 days
```

Comment:
we can decrease this further to 2k, to be same as scanBatchRequests, since we have 1-1 call since we dont expect more than 1 request per batch request from customer